core/roboconf-dm-templating/src/main/java/net/roboconf/dm/templating/internal/templates/TemplateWatcher.java

```diff
@@ -0,0 +1,378 @@
+/**
```

Comment:
I think you messed up with git here ;)
File should have been moved, not deleted + new file at new location. Same for TemplateEntry...